### DISCUSSION TITLE
Translation XLIFF export/import bug once a document was translated?

### DISCUSSION THREAD

**Discussion by akno77** [pimcore/pimcore#15575] [Q&A / Support]
Hi,

I'm currently struggling with the XLIFF export/import tool.

If I copy a existing page to a new language version and export the XLIFF for the new page, I get a XLIFF file with correct source/target values (source and target are the same, since all has to be translated). The import also works fine, the translated values appear correctly.

The problem is, if I make new text changes to the source document and export the target document again, the translation values for source & target are the same (= values of target document).

Is this a bug or a feature? I'd expect that the translation value for the source comes from the source document and not from the target document...

Used Pimcore version: v11.0.3

**Comment by plixus** [comment]
Hi @akno77, I am currently dealing with XLIFF translations as well and so far everything seems to work. 
Can you maybe post some screenshots so I can better understand the problem?
You write that Source and Target are the same on the first export, that sounds kind of wrong ...

**Reply by akno77** [reply]
Hi @plixus, sorry for the late answer.

I still have the problem with Pimcore v11.0.9.

Here are some screenshots.

The German language version document:
![grafik](https://github.com/pimcore/pimcore/assets/35833873/474f10a9-7062-4640-b3a3-a62aea25f80f)

The English language version document:
![grafik](https://github.com/pimcore/pimcore/assets/35833873/2f570244-10b9-4b7e-bf86-4be43ca13c42)

The export of the German language document with source = EN and traget = DE
![grafik](https://github.com/pimcore/pimcore/assets/35833873/56fc7398-50eb-4dde-8c0a-da7845ccaf79)

The exported XLIFF file in poedit, the source and target are the same:
![grafik](https://github.com/pimcore/pimcore/assets/35833873/44ed7e88-ad15-4d99-8b41-ed71dc8c836d)

**Reply by akno77** [reply]
`You write that Source and Target are the same on the first export, that sounds kind of wrong ...`

I mean the translation values are the same on the first export which is correct, because you copy the English document to german then you have the same translations initially.

**Reply by akno77** [reply]
I created abug report for this: https://github.com/pimcore/pimcore/issues/16037
